pkg/moov/transfer_api.go

```diff
@@ -46,23 +38,51 @@ func (c Client) CreateTransfer(ctx context.Context, transfer CreateTransfer, opt
 		callArgs = append(callArgs, opt(builder))
 	}
 
-	resp, err := c.CallHttp(ctx, Endpoint(http.MethodPost, pathTransfers), callArgs...)
+	return CreateTransferBuilder{
+		client:   c,
+		ctx:      ctx,
+		endpoint: Endpoint(http.MethodPost, pathTransfers),
+		callArgs: callArgs,
+	}
+}
+
+type CreateTransferBuilder struct {
```

Comment:
This probably isn't the best name since it can be confused with `CreateTransfer` param...